packages/jellyfish-api-core/__tests__/category/masternode/getMasternodeBlock.test.ts

```diff
@@ -0,0 +1,143 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { RpcApiError } from '../../../src'
+
+describe('Masternode', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should getMasternodeBlocks', async () => {
+    const masternodeID = await client.masternode.createMasternode(await client.wallet.getNewAddress())
```

Comment:
Use `container.call` like what Webster mentioned in your another PR

Reply:
Remember `container.call` fails on masternode rpcs? The only option here is to use `container.getnewaddress`